### PR TITLE
Contributor

### PR DESCRIPTION
**Pull request by savagecarol**
# Related Issue
- #42 

# Proposed changes
- make a function in about.dart from the api 
- shifting of some file
- removal of unused code in main.dart

# Description
- main.dart issues is also fixed
- theme file shifted in services folder
- about page is working perfectly

# Screenshots
Screenshot_1607415998

### CODE REVIEW

File: lib/home.dart [hunk 1]
```diff
@@ -54,19 +51,19 @@ class _HomeState extends State<Home> with SingleTickerProviderStateMixin {
...
                     ),
                     Switch(
                         value: _themeChanger.lightDark,
-                        onChanged: (lightDark){
-                          if(lightDark)
-                         _themeChanger.setTheme(ThemeData.light());
-                         else
-                          _themeChanger.setTheme(ThemeData.dark());
-
-
```
Comment (harry-hov) [inline lib/home.dart:69]: This PR is all about contributors page. Try to address single issue in a single PR.

You can always send multiple PRs for multiple issues. ;)

File: lib/about.dart [hunk 2]
```diff
@@ -1,76 +1,74 @@
...
-              "© enzaimz",
-              
-              style: new TextStyle(
-                fontSize: 20.0,
-                fontWeight: FontWeight.w200,
-              ),
-            ),
-          ),
-        ],
+        title: Text('About Developers'),
```
Comment (harry-hov) [inline lib/about.dart:51]: 'Contributors' seems better to me.

File: lib/about.dart [hunk 3]
```diff
@@ -1,76 +1,74 @@
...
       ),
+     body: isLoading
+          ? Center(child: CircularProgressIndicator())
+          : ListView.builder(
+              itemCount: names.length,
+              itemBuilder: (context, index) {
+                return Card(
+                  child: Container(
+                    padding: EdgeInsets.all(20.0),
+                    child: Link(
```
Comment (harry-hov) [inline lib/about.dart:61]: Can we use ListTile inside card?
Something like:
```
ListTile(
    title: Text(Name),
    subtitle: Text('GitHub: ' + username),
),
```

Comment (savagecarol) [reply lib/about.dart:61]: i put the list tile 

Screenshot_1607418815

File: lib/home.dart (hunk 1 above)
Comment (savagecarol) [reply lib/home.dart:69]: okay i will keep in mind for next time

File: lib/about.dart (hunk 2 above)
Comment (savagecarol) [reply lib/about.dart:51]: i will improve my code

File: lib/about.dart (hunk 3 above)
Comment (harry-hov) [reply lib/about.dart:61]: It looks good. Can you also fetch name?

**Comment by savagecarol** [comment]
{
    "login": "ashiscs",
    "id": 49238464,
    "node_id": "MDQ6VXNlcjQ5MjM4NDY0",
    "avatar_url": "https://avatars1.githubusercontent.com/u/49238464?v=4",
    "gravatar_id": "",
    "url": "https://api.github.com/users/ashiscs",
    "html_url": "https://github.com/ashiscs",
    "followers_url": "https://api.github.com/users/ashiscs/followers",
    "following_url": "https://api.github.com/users/ashiscs/following{/other_user}",
    "gists_url": "https://api.github.com/users/ashiscs/gists{/gist_id}",
    "starred_url": "https://api.github.com/users/ashiscs/starred{/owner}{/repo}",
    "subscriptions_url": "https://api.github.com/users/ashiscs/subscriptions",
    "organizations_url": "https://api.github.com/users/ashiscs/orgs",
    "repos_url": "https://api.github.com/users/ashiscs/repos",
    "events_url": "https://api.github.com/users/ashiscs/events{/privacy}",
    "received_events_url": "https://api.github.com/users/ashiscs/received_events",
    "type": "User",
    "site_admin": false,
    "contributions": 1
  },



there is no name field.... 
instead of that you can put avatar

**Comment by harry-hov** [comment]
Hey @savagecarol, you are doing great with your coding skills.

But one thing I would like to suggest is please also focus on your commit title and commit message.

**Comment by harry-hov** [comment]
> there is no name field....

oh! Didn't knew that.

> instead of that you can put avatar

If you want you can try adding avatar.

**Comment by harry-hov** [comment]
Also, please squash your commits in a single commit.

**Comment by savagecarol** [comment]
okay

**Comment by savagecarol** [comment]
avatar part is completed
Screenshot_1607424257

**Comment by savagecarol** [comment]
i pushed the code on the ---  contributor branch

**Comment by harry-hov** [comment]
Nicely done. Can you please squash all of your commits in a single commit.

**Comment by savagecarol** [comment]
okay cool

**Comment by savagecarol** [comment]
check it now

**Comment by savagecarol** [comment]
in my gitlog only one commit showing

**Comment by abhitidarbar** [comment]
> in my gitlog only one commit showing

For now your pr is merged so that you can have start focusing on other issues but for next time keep in mind to squash your commit into single one .Also, try to use self explanatory commit messages about what that commit is all about rather than "all done". 
However ,your work and efforts are appreciated 👍 .

**Pull request merged**